src/app/rooms/admin/room-form/room-form.component.css

```diff
@@ -14,3 +14,471 @@ pre {
 small {
   display: none;
 }
+
+
+
+.datepicker {
```

Comment:
If you remove the old datepicker, remove these styles.

But in case you decide to keep the old datepicker, move these styles to this file https://github.com/phyo-kyaw/break-booking-frontend/blob/master/src/styles.css